spec/spec.md

```diff
@@ -748,6 +723,98 @@ where:
 
 The wallet endpoint should present the provided messages for approval. If approved, the wallet endpoint should sign the messages with the private key for the authorized account address, and return the signed messages to the dapp endpoint. The signatures should be appended to the message, in the same order as `addresses`.
 
+#### sign_in_with_solana
```

Comment:
I think this should be moved to a non-privileged method. The idea with SIWS (in wallet-standard) is that it combines connect + sign message and returns the account that was used to sign the payload (along with the signature). MWA differs from web-wallets in that we have a session that authorizes a set of accounts for use within the session. I think Sign in With Solana should effectively be a way to establish an authorized session with a single account.